substrate/primitives/metadata-ir/src/v14.rs

```diff
@@ -151,7 +151,7 @@ impl From<ExtrinsicMetadataIR> for ExtrinsicMetadata {
 	fn from(ir: ExtrinsicMetadataIR) -> Self {
 		ExtrinsicMetadata {
 			ty: ir.ty,
-			version: ir.version,
+			version: *ir.versions.get(0).expect("Metadata V14 supports only one version"),
```

Comment:
Is there any plan to support several versions here and how to to find the latest version? I guess it will be last item in the Vec?

So my question is really if you need to do a linear search to find the specific version?

Reply:
Agreed that V14 should continue to return the lowest version supported.